pkg/util/util.go

```diff
@@ -144,7 +144,14 @@ func Copy(m *clusterv1.Machine) *clusterv1.Machine {
 }
 
 func ExecCommand(name string, args ...string) string {
-	cmdOut, _ := exec.Command(name, args...).Output()
+	cmdOut, err := exec.Command(name, args...).Output()
+	if err != nil {
+		s := name
+		if len(args) != 0 {
+			s += " " + strings.Join(args, " ")
+		}
```

Comment:
What about something like `strings.Join(append([]string{name}, args...), " ")`?

Reply:
Thanks - made that change!